Provide a way to enter a line total and have the quantity sold automatically calculated.

SteveIreland opened the pull request:
This should result in the behavior last discussed.

daN4cat commented on application/views/sales/register.php:
```diff
@@ -179,11 +179,11 @@
 								}
 								else
 								{
-									echo to_currency($item['price']*$item['quantity']-$item['price']*$item['quantity']*$item['discount']/100);
```
I'm afraid I cannot get where this is now calculated. Your changes don't show this calculation being moved somewhere else. Or am I missing something?

SteveIreland replied:
Sale_lib.php line 903

$line['discounted_total'] = $this->get_item_total($quantity, $price, $discount, TRUE);

daN4cat replied:
Ok, got the point

SteveIreland commented:
Okay, I've removed the statements that were attempting to check for a required field on the extended total.  To have properly corrected it I would have needed to add an item info retrieval to the method.  I didn't think it was worth the overhead so I just removed the code.  If the user inexplicably enters a zero amount then the quantity will be set to zero. I think it will be okay.
